private/bufpkg/bufconnect/interceptors_test.go

```diff
@@ -147,3 +151,21 @@ func TestCLIWarningInterceptorFromError(t *testing.T) {
 	assert.Error(t, err)
 	assert.Equal(t, fmt.Sprintf("WARN\t%s\n", warningMessage), buf.String())
 }
+
+func TestNewAugmentedConnectErrorInterceptor(t *testing.T) {
+	t.Parallel()
+	mux := http.NewServeMux()
+	mux.Handle(modulev1connect.NewCommitServiceHandler(&modulev1connect.UnimplementedCommitServiceHandler{}))
+	server := memhttptest.New(t, mux)
+	client := modulev1connect.NewCommitServiceClient(
+		server.Client(),
+		server.URL(),
+		connect.WithInterceptors(NewAugmentedConnectErrorInterceptor()),
+	)
+	_, err := client.GetCommits(context.Background(), connect.NewRequest(&modulev1.GetCommitsRequest{}))
+	assert.Error(t, err)
+	var augmentedConnectError *AugmentedConnectError
+	assert.ErrorAs(t, err, &augmentedConnectError)
+	assert.Equal(t, "example.com", augmentedConnectError.Addr())
+	assert.Equal(t, "/buf.registry.module.v1.CommitService/GetCommits", augmentedConnectError.Procedure())
```

Comment:
I _think_ I need a "real" server/client to populate the addr and procedure here; not sure if there's a way to do this better.